app/services/auth.js

```diff
@@ -116,9 +116,11 @@ export default Service.extend({
 
       Travis.trigger('user:signed_in', this.currentUser);
 
-      this.reloadCurrentUser().then(() =>
-        Travis.trigger('user:refreshed', data.user)
-      );
+      this.reloadCurrentUser().then(() => {
+        Travis.trigger('user:refreshed', data.user);
+        if (this.currentUser && this.currentUser.get('recentlySignedUp') && this.currentUser.get('recentlySignedUp') === true)
+          this.metrics.trackPage({ page: '/first_sync' });
```

Comment:
This doesn't look right to me. 

1) Wouldn't it report `/first_sync` page visit twice for new users? First one - here, second one - after actual redirect to the `first_sync` page?

2) I'd suggest to use `reportNewUser` method instead, as it's already in place (see below) and aimed for the same purpose. You could just change it as necessary and invoke wherever it work best.